src/index.html

```diff
@@ -10,13 +10,100 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+  <header class="header">
+    <a
+      href="#home"
+      class="header__link"
+    >
+      <img
+        class="img"
+        src="./images/logo.png"
+        alt="moyo_logo"
+      />
+    </a>
+    <nav class="nav">
+      <ul class="nav__list">
+        <li class="nav__item">
+          <a
+            href="#"
+            class="nav__link is-active"
+          >
+            Apple
+          </a>
+        </li>
+        <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
        </li>
        
        <li class="nav__item">
```